### PR TITLE
Support Accept/Reject/Next/Prev chat edit actions in notebooks

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/vs/workbench/contrib/chat/browser/chatEditorActions.ts [hunk 1]
```diff
@@ -36,7 +38,7 @@ abstract class NavigateAction extends Action2 {
 					? KeyMod.Alt | KeyCode.F5
 					: KeyMod.Alt | KeyMod.Shift | KeyCode.F5,
 				weight: KeybindingWeight.EditorContrib,
-				when: ContextKeyExpr.and(ctxHasEditorModification, EditorContextKeys.focus),
+				when: ContextKeyExpr.and(ContextKeyExpr.or(ctxHasEditorModification, ctxNotebookHasEditorModification), EditorContextKeys.focus),
```
Comment (DonJayamanne) [inline src/vs/workbench/contrib/chat/browser/chatEditorActions.ts:41]: The `ctxHasEditorModifcations` only applies to regular editors & not notebooks, hence the new context var

**Comment by DonJayamanne** [comment]
@jrieken Making minimal changes in chat edits, i.e. trying to limit changes before we start baking in full (native) support for notebooks in chat edits. However I believe these changes are still valid
Merging into a separate branch for now so you can just see the changes specific to chat edits area.

File: src/vs/workbench/contrib/chat/browser/chatEditorActions.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/vs/workbench/contrib/chat/browser/chatEditorActions.ts:41]: Alternatively I can create new menu items and ensure these are disabled for notebooks.

File: src/vs/workbench/contrib/chat/browser/chatEditorActions.ts [hunk 2]
```diff
@@ -144,20 +146,32 @@ abstract class AcceptDiscardAction extends Action2 {
...
+				return notebookEditor.textModel?.uri;
+			}
+			const editor = editorService.activeTextEditorControl;
+			if (!isCodeEditor(editor) || !editor.hasModel()) {
+				return;
+			}
+			return editor.getModel().uri;
+		};
+
+		const uri = getUri();
```
Comment (jrieken) [inline src/vs/workbench/contrib/chat/browser/chatEditorActions.ts:161]: Can we just compute this variable without defining and calling the function? Some good old `let uri: Uri|undefined...` and some if/else

**Review by jrieken — DISMISSED** [review]

**Review by bpasero — APPROVED** [review]

**Pull request merged**